src/main/java/tasks/Event.java

```diff
@@ -0,0 +1,41 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * For the creation of an Event class
+ */
+public class Event extends Task {
+
+    protected LocalDate start;
```

Comment:
do you think you should name these startDate and toDate respectively?

Reply:
I do think so! I'll be making the adjustments promptly. Also, as a general note, I'll likely be implementing most of the changes you listed, just didn't want to comment on every single comment, haha. Thanks very much for the review!